src/utils/editor-environment.js

```diff
@@ -2,11 +2,13 @@
  * Internal dependencies
  */
 import { awaitGBKitGlobal, editorLoaded, getGBKit } from './bridge';
+import { configureLocale } from './localization';
```

Comment:
There is no need to lazy load this module as its WordPress dependencies are loaded via the script tag in the entry `index.html` and therefore the relevant `window.wp` globals are already defined at this point.